dags/maxtext_pathways/configs/utils.py

```
@@ -188,3 +275,237 @@ def check_gcp_logs_exist(
+  ti = kwargs["task_instance"]
+
+  with tempfile.NamedTemporaryFile() as temp_config_file:
+    env = _authenticate_kubectl(
+        project_id, region, cluster_name, kubeconfig_path=temp_config_file.name
+    )
+    pods = _list_workload_pods_kubectl(workload_id, env=env)
+
+    if not pods:
+      raise AirflowException(
+          f"No pods found for workload selector: {workload_id}"
+      )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)